.golangci.yml

```diff
@@ -12,6 +12,7 @@ linters:
     - gofmt
     - govet
     - deadcode
+    - depguard
```

Comment:
why isn't there a linter that just checks for deprecated functions in stdlib?

Reply:
Because ...  _"do as we say, not as we do"_ (?) they didn't deprecated it in code; https://github.com/golang/go/blob/go1.16.15/src/io/ioutil/ioutil.go#L25-L26

Otherwise `staticcheck` would've picked it up (e.g. like for `strings.Title()` (which IMO shouldn't have been marked deprecated, just had its limitations better documented); see https://github.com/moby/moby/pull/43201